driver_test.go

```diff
@@ -1914,6 +1928,91 @@ func startDestroyInspectImage(t *testing.T, image string, taskName string) {
 	require.Equal(t, imageID, inspectData.Image)
 }
 
+// TestPodmanDriver_Pull_Timeout verifies that the task image_pull_timeout
+// configuration is respected and that it can be set to higher value than the
+// driver's client_http_timeout.
+//
+// This test starts a proxy on port 5000 and requires the machine running the
+// test to allow an insecure registry at localhost:5000. To run this test add
+// the following to /etc/containers/registries.conf:
+//
+// [[registry]]
+// location = "localhost:5000"
+// insecure = true
+func TestPodmanDriver_Pull_Timeout(t *testing.T) {
```

Comment:
I have no idea how to test this reliably 🤷 

I couldn't find any documentation on how to make a Podman API request to ignore insecure TLS so this needs to be configured in the host.

Reply:
This is the correct way to do an insecure registry. I use `insecure = true` when firing up an ephemernal registry for local development.

Reply:
Thanks! I was looking for a way to test this that wouldn't require a specific setting on the local machine, but I couldn't figure out a way, so that's the best we have for now 😅